OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/CourseUnitPagerAdapter.java

```diff
@@ -110,6 +110,7 @@ else if (isCourseUnitVideo(minifiedUnit)) {
         } else if (minifiedUnit.getType() != BlockType.VIDEO &&
                 minifiedUnit.getType() != BlockType.HTML &&
                 minifiedUnit.getType() != BlockType.OTHERS &&
+                minifiedUnit.getType() != BlockType.OPENASSESSMENT &&
```

Comment:
Same comment as before, why below OTHERS this type and not at the end?

Reply:
Moved it to the end